src/main/java/org/snomed/quality/validator/mrcm/Application.java

```diff
@@ -1,59 +1,64 @@
 package org.snomed.quality.validator.mrcm;
 
+import org.ihtsdo.otf.snomedboot.ReleaseImportException;
+import org.ihtsdo.otf.sqs.service.exception.ServiceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.NotDirectoryException;
 import java.text.SimpleDateFormat;
+import java.util.Arrays;
 import java.util.Collections;
+import java.util.List;
 
-import org.ihtsdo.otf.snomedboot.ReleaseImportException;
-import org.ihtsdo.otf.sqs.service.exception.ServiceException;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import static org.snomed.quality.validator.mrcm.Constants.*;
 
 public class Application {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(Application.class);
-	
-	public static void main(String[] args) throws Exception {
-		SimpleDateFormat dateFormat = new SimpleDateFormat("yyyyMMdd");
-		String releasePackage = null;
-		String releaseDate = null;
-		File resultDir = null;
-		boolean isStatedViewOnly = true;
-		if ( args == null || args.length < 4) {
-			String msg = "Please specify the java arguments after replacing the {} with actual values." + "{release_package_unzipped_root} {is_stated_only} {release_date} {result_dir}";
-			System.out.println(msg);
-			System.out.println("{release_package_unzipped_root_dir} is for the release package unzipped file root directory. eg: /Users/Releases/SnomedCT_InternationalRF2_PRODUCTION_20170731T120000Z");
-			System.out.println("{is_stated_only} is to state whether to use stated relationships only. This parameter is optional and the default value is set to true.");
-			System.out.println("{release_date} is the effective date for the release file that is being validated.The format is yyyyMMdd eg:20170731");
-			System.out.println("{result_dir} is the directory where validation reports will be saved.");
-			throw new IllegalStateException(msg);
+
+	public static void main(final String[] args) throws Exception {
+		if (args == null || args.length < 4) {
+			LOGGER.info(ERROR_MESSAGE);
+			LOGGER.info(RELEASE_PACKAGE_UNZIPPED_ROOT_DIR_HELP_MESSAGE);
+			LOGGER.info(IS_STATED_ONLY_HELP_MESSAGE);
```

Comment:
Use Logger instead of System.out looks tidy. And we should always use logger in normal java class. However for stand-alone main class I think it is ok.
It looks the java argument for '{is_stated_ony}' is changed as it is not boolean value. Instead of "stated" or "inferred". I would expect this message is updated to reflect this.

Reply:
Fixed.